python/python/glide/async_commands/core.py

```diff
@@ -2262,3 +2262,32 @@ async def invoke_script(
                 ["foo", "bar"]
         """
         return await self._execute_script(script.get_hash(), keys, args)
+
+    async def pfadd(self, key: str, elements: List[str]) -> int:
+        """
+        Adds all elements to the HyperLogLog data structure stored at the specified `key`.
+        Creates a new structure if the `key` does not exist.
+
+        When no `elements` are provided, and `key` exists and is a HyperLogLog, then no operation is performed.
+        If `key` does not exist, then the HyperLogLog structure is created.
+
+        See https://redis.io/commands/pfadd/ for more details.
+
+        Args:
+            key (str): The `key` of the HyperLogLog data structure to add elements into.
+            elements (List[str]): An list of members to add to the HyperLogLog stored at `key`.
+
+        Returns:
+            int: If the HyperLogLog is newly created, or if the HyperLogLog approximated cardinality is
+            altered, then returns `1`. Otherwise, returns `0`.
```

Comment:
```suggestion
            altered, then returns 1. Otherwise, returns 0.
```

I dont remember doing that in other commands in python

Reply:
We do that in java for better readability